### ISSUE TITLE
README + DEMO

### ISSUE THREAD

**Issue by ayoubelmhamdi** [bashbunni/cli-chat#2]
@bashbunni , could you add some demos and descriptions, of how these tools look and work?

**Comment by bashbunni** [comment]
Yep! Can do once it's working. I'll leave this issue open until then

**Comment by KunalKumar-1** [comment]
is it finished ? 
looking out for a little doc/readme for guidance and usage
